src/SprykerEco/Shared/UnzerApi/Transfer/unzer_api.transfer.xml

```diff
@@ -0,0 +1,456 @@
+<?xml version="1.0"?>
```

Comment:
Does not match the definition from the tech spec https://spryker.atlassian.net/wiki/spaces/CORE/pages/2802745361/WIP+CC-14082+SprykerEco+Unzer
Please adjust the tech spec(or code).